src/app/3d/qgs3dmapconfigwidget.cpp

```diff
@@ -75,6 +75,7 @@ Qgs3DMapConfigWidget::Qgs3DMapConfigWidget( Qgs3DMapSettings *map, QgsMapCanvas
   spinMapResolution->setClearValue( 512 );
   spinScreenError->setClearValue( 3 );
   spinGroundError->setClearValue( 1 );
+  terrainElevationOffsetSpinBox->setClearValue( 0.0f );
```

Comment:
```suggestion
  terrainElevationOffsetSpinBox->setClearValue( 0.0 );
```

(The widget uses double values, not float)